### DISCUSSION TITLE
Failed to locate 'clang'

### DISCUSSION THREAD

**Discussion by GeneWilburn** [Homebrew/discussions#3874] [Getting started] [answered]
When updating Homebrew on an M1 Mac Mini running Ventura 13, I keep getting this message even though I've installed the latest command line utilities.

$ which clang
/usr/bin/clang

I use Homebrew's Bash as my shell rather than zshell.

Does anyone have any suggestions?

Thanks,

Gene

**Comment by SMillerDev** [comment]
What is the message?

**Reply by GeneWilburn** [reply]
The message is "Failed to locate 'clang'"

Clang is on the system located at /usr/bin/clang and the latest MacOS command-line tools have been installed. I’m unable to successfully run 'brew upgrade'

**Reply by SMillerDev** [reply]
Can you post the full output you see? Because Homebrew doesn't call clang unless you're installing something from source.

**Comment by carlocab** [comment, marked as answer]
What is the output of `brew config` and `brew doctor`?

**Reply by GeneWilburn** [reply]
Thanks. Running `brew doctor` showed a conflict with Xcode, suggesting I update or delete it. After I deleted Xcode, I ran `brew doctor` again and it responded with "Your system is ready to brew."
